.gitignore

```diff
@@ -25,3 +25,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 storybook-static
+.yalc
+yalc.lock
```

Comment:
What is `yalc`?

Reply:
`yalc` solves the same problem as `yarn link/npm link`, but it actually copies deps and maintains a sort of local package registry.

https://github.com/whitecolor/yalc

It works more like real dependencies, and, in this case, solved some issues with peer deps when I was trying to test local front-ends importing local RoverUI.

However, I haven't figured out the best options to use it with, so I had to let it make changes to my front-end code's package.json file. Not ideal, long term, so I haven't documented it as a "solution" for local testing.

I wanted to add it to .gitignore, because I anticipate using it for testing again and this reduces the risk of me pushing local registry metadata to git.